### ISSUE TITLE
Follow specific thread

### ISSUE THREAD

**Issue by RayBB** [flytaly/reddit-post-notifier#6]
Is it possible to have a setup where I can follow for comments on a specific thread?

Kind of like how you can turn on inbox replies for your own posts?

The use case is if I see a thread that I like and wanna be updated on if someone comments.

Thanks

**Comment by flytaly** [comment]
No, you can't follow for comments currently.  But there is [API for comments](https://www.reddit.com/dev/api/oauth#GET_comments_{article}), so maybe I will add this option later.

As a workaround, you can use rss endpoint with any RSS reader. There should be browser extensions for RSS too.

A thread for example: `https://www.reddit.com/r/Frontend/comments/hdpu09/making_sense_of_redux/`.

Add `comments.rss?sort=new` at the end or `comments.rss?sort=new&depth=1` if you need only top level comments.
So the link now looks like this:
`https://www.reddit.com/r/Frontend/comments/hdpu09/making_sense_of_redux/comments.rss?sort=new`
and you can put it into RSS-reader.

**Comment by dandepeched** [comment]
@flytaly +1 for this feature. Reddit RSS feed is unreliable, so new comments are missing, ?sort=new makes results even worse :(

**Comment by flytaly** [comment]
Reddit now has the built-in feature to follow a post. It's only available in the new design.
![follow posts](https://user-images.githubusercontent.com/1577569/147551960-bed4f696-f976-44e6-bbbe-43e3f0bc4aa2.png)

Though it looks like it works only with level 1 comments, and you need to manually subscribe to every comments down the comment tree with **Follow** button.
![follow comment](https://user-images.githubusercontent.com/1577569/147552159-c95f3bb4-3aa8-4e8c-bc1f-e75cabc30941.png)

Even if I implement it in the extension, it would work the same way as RSS/Reddit built-in feature works. Reddit has an old API, and it's impossible to efficiently fetch new nested replays. You can only fetch level 1 replays to a post or comment.

In other words, I can add follow thread functionally, but it will work only with level 1 comments.

**Comment by dandepeched** [comment]
@flytaly , Reddit solution have significant delay. Will it be possible to create same notification type but with adjustable delay?
